app/components/dex-create.jsx

```diff
@@ -71,13 +80,28 @@ export class DexCreate extends Component {
             </div>
             <div className="form-group">
               <label htmlFor="generation">Generation</label>
-              <select className="form-control" ref={(c) => this._generation = c} defaultValue="7">
+              <select className="form-control" onChange={this.handleChange} value={this.state.generation}>
                 <option value="7">Seven</option>
                 <option value="6">Six</option>
               </select>
               <i className="fa fa-chevron-down" />
             </div>
             <div className="form-group">
+              <label htmlFor="region">Regionality</label>
+              <div className="radio">
+                <label>
+                  <input type="radio" name="region" checked={region === 'national'} value="national" onChange={() => this.setState({ ...this.state, region: 'national' })} />
+                  <span className="radio-custom"><span /></span>National
+                </label>
+              </div>
+              <div className={`radio ${generation === '6' ? 'disabled' : ''}`}>
+                <label>
+                  <input type="radio" name="region" checked={region === 'alola'} disabled={generation === '6'} value="alola" onChange={() => this.setState({ ...this.state, region: 'alola' })} />
```

Comment:
i thought about it, but wasn't sure what to put though? Could be something like `Not yet supported` but that doesn't give you any more information than what the disabled state already does, ya know? but not opposed to it.